tensorrt_llm/_utils.py

```
@@ -1192,29 +1196,37 @@ def confidential_compute_enabled() -> bool:
+                      == pynvml.NVML_CC_SYSTEM_FEATURE_ENABLED
+                      or cc_settings.multiGpuMode
+                      == pynvml.NVML_CC_SYSTEM_MULTIGPU_PROTECTED_PCIE)
+        nvle_enabled = (
+            cc_settings.multiGpuMode == pynvml.NVML_CC_SYSTEM_MULTIGPU_NVLE)
     except pynvml.NVMLError_NotSupported:
         # Simple query for older GPUs
         try:
             cc_state = pynvml.nvmlSystemGetConfComputeState()
             cc_enabled = (
                 cc_state.ccFeature == pynvml.NVML_CC_SYSTEM_FEATURE_ENABLED)
         except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)